test/dynamo/test_misc.py

```diff
@@ -1278,21 +1278,59 @@ def fn(x, y):
             self.assertTrue(same(ref, res))
         self.assertEqual(cnts.frame_count, 2)
 
-    def test_(self):
-        def fn(x: torch.Tensor, y: int):
-            z = x.detach()
-            w = y + 1
-            torch._dynamo.graph_break()
-            return z + w
+    @requires_numpy_pytorch_interop
+    def test_mandelbrot_numpy(self):
+        def mandelbrot_numpy(max_iter):
+            # Define the boundaries of the complex plane
+            xn = 450
+            yn = 375
+            xmin = -2.25
+            xmax = 0.75
+            ymin = -1.25
+            ymax = 1.25
+
+            # Create the grid of complex numbers
+            x_values = np.linspace(xmin, xmax, xn, dtype=np.float64)
+            y_values = np.linspace(ymin, ymax, yn, dtype=np.float64)
+            rx, iy = np.meshgrid(x_values, y_values, indexing="xy")
+
+            x = rx.copy()
+            y = iy.copy()
+            mask = np.zeros_like(x)
+            for i in range(max_iter):
+                x_prev = x
+                y_prev = y
+                x = x_prev**2 - y_prev**2 + rx
+                y = 2 * x_prev * y_prev + iy
+                inside = np.sqrt(x**2 + y**2) <= 2
+                mask += inside
+            return mask
 
         cnts = torch._dynamo.testing.CompileCounter()
-        opt_fn = torch._dynamo.optimize(cnts)(fn)
+        opt_fn = torch._dynamo.optimize(cnts, nopython=True)(mandelbrot_numpy)
         for _ in range(10):
-            x = torch.randn([1, 3])
-            y = 5
+            x = torch.randint(100, (1,))
+            ref = mandelbrot_numpy(x.item())
+            res = opt_fn(x.item())
+            self.assertTrue(same(ref, res))
```

Comment:
How many frames is this in compile counter?

Reply:
nopython=True so 1?

Reply:
Or 0 ;) No frames because no compiles will still pass with nopython=True AND will give a correct response on result comparison.

Reply:
actually it creates multiple graphs because of the for loop. Not too sure what happens here